api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -102,6 +121,7 @@ message KubernetesClusterMetadata {
 message SendKubernetesResourcesRequest {
   // Send any of these "request" messages that can contain any of the documented types.
   oneof request {
+    Keepalive keepalive = 5;
```

Comment:
I don't understand why you'd need to partially pull changes from unmerged PR #155.
Revert this.
```suggestion
```